.github/workflows/load-tests-job-configs/config_GBK_Flink_Go_Batch_Reiteration_10KB.txt

```diff
@@ -0,0 +1,30 @@
+###############################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+--job_name=load-tests-go-flink-batch-gbk-6-
+--influx_namespace=flink
+--influx_measurement=go_batch_gbk_6
+--iterations=4
+--fanout=1
+--parallelism=5
+--input_options=''{\"num_records\":20000000,\"key_size\":10,\"value_size\":90,\"num_hot_keys\":200,\"hot_key_fraction\":1}''
+--endpoint=localhost:8099
+--environment_type=DOCKER
+--environment_config=gcr.io/apache-beam-testing/beam-sdk/beam_go_sdk:latest
+--influx_db_name=beam_test_metrics
+--influx_hostname=http://10.128.0.96:8086
```

Comment:
same as https://github.com/apache/beam/pull/28450#discussion_r1333171697 a load test builder step to set common options (e.g., currently if influxHost has changed then it needs to change all files)

Reply:
Done